packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-modal-checkbox.spec.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import JurisdictionModalCheckbox from '../jurisdiction-modal-checkbox';
+import RootStore from 'Stores/index';
+
+describe('JurisdictionModalCheckbox', () => {
+    const mock_store = {
+        client: {},
+        common: {},
+        ui: {},
+    };
+    const mockRootStore = new RootStore(mock_store);
+
+    const mock_props = {
+        class_name: '',
+        is_checked: false,
+        jurisdiction_selected_shortcode: '',
+        onCheck: jest.fn(),
+        context: mockRootStore,
+        should_restrict_bvi_account_creation: false,
+        should_restrict_vanuatu_account_creation: false,
+    };
+
+    const labuan_account = {
+        class_name: 'cfd-jurisdiction-checkbox',
+        is_checked: false,
+        jurisdiction_selected_shortcode: 'labuan',
+        onCheck: jest.fn(),
+        context: mockRootStore,
+        should_restrict_bvi_account_creation: false,
+        should_restrict_vanuatu_account_creation: false,
+    };
+
+    const svg_account = {
+        class_name: 'cfd-jurisdiction-checkbox',
+        is_checked: false,
+        jurisdiction_selected_shortcode: 'svg',
+        onCheck: jest.fn(),
+        context: mockRootStore,
+        should_restrict_bvi_account_creation: false,
+        should_restrict_vanuatu_account_creation: false,
+    };
+
+    const bvi_account_without_restriction = {
+        class_name: 'cfd-jurisdiction-checkbox',
+        is_checked: false,
+        jurisdiction_selected_shortcode: 'bvi',
+        onCheck: jest.fn(),
+        context: mockRootStore,
+        should_restrict_bvi_account_creation: false,
+        should_restrict_vanuatu_account_creation: false,
+    };
+
+    const bvi_account_with_restriction = {
+        class_name: 'cfd-jurisdiction-checkbox',
+        is_checked: false,
+        jurisdiction_selected_shortcode: 'bvi',
+        onCheck: jest.fn(),
+        context: mockRootStore,
+        should_restrict_bvi_account_creation: true,
+        should_restrict_vanuatu_account_creation: false,
+    };
+
+    const vanuatu_account_without_restriction = {
+        class_name: 'cfd-jurisdiction-checkbox',
+        is_checked: false,
+        jurisdiction_selected_shortcode: 'vanuatu',
+        onCheck: jest.fn(),
+        context: mockRootStore,
+        should_restrict_bvi_account_creation: false,
+        should_restrict_vanuatu_account_creation: false,
+    };
+
+    const vanuatu_account_with_restriction = {
+        class_name: 'cfd-jurisdiction-checkbox',
+        is_checked: false,
+        jurisdiction_selected_shortcode: 'vanuatu',
+        onCheck: jest.fn(),
+        context: mockRootStore,
+        should_restrict_bvi_account_creation: false,
+        should_restrict_vanuatu_account_creation: true,
```

Comment:
we can remove all these constants and use mockprops as below. 
render(<JurisdictionModalCheckbox {...mock_props} jurisdiction_selected_shortcode: 'labuan', />);